src/backend/migrations/20221101184712_geese-features.js

```diff
@@ -0,0 +1,9 @@
+exports.up = (knex) =>
+  knex.schema.createTable('geese-features', (table) => {
+    table.increments('id');
+    table.string('name');
+    table.string('picture');
+    table.string('description');
```

Comment:
Let's add `.notNullable()` to the 3 string fields to maintain consistency with your API design :)